lib/widgets/popular_job_card_builder.dart

```diff
@@ -15,13 +15,13 @@ class PopularJobCardBuilder extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Container(
-      padding: EdgeInsets.all(15),
+      padding: const EdgeInsets.all(15),
       height: 160,
       width: 260,
       decoration: BoxDecoration(
         color: Theme.of(context).colorScheme.tertiary,
         borderRadius: BorderRadius.circular(20),
-        boxShadow: [
+        boxShadow: const [
           BoxShadow(
             color: Color(0x1A000000),
```

Comment:
have a colors.dart file and have all colors there. name them properly on what color are they.